README.md

```diff
@@ -26,13 +26,17 @@ To run that demo on your own computer:
 
 ## How to Use
 
-Ensure packages are installed with correct version numbers by running:
+Ensure packages are installed with correct version numbers by running the following shell script: 
   ```sh
   (
     export PKG=react-dates;
     npm info "$PKG" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g; s/ *//g' | xargs npm install --save "$PKG"
   )
   ```
+If you're new to shell scripts, create an install.sh file, copy the command above to it and run by typing this in your terminal: 
```

Comment:
@ljharb not sure if you saw it, but I put together a powershell script that should get the trick done - it's in the comments here: https://github.com/airbnb/react-dates/pull/107